src/scss/Landing.scss

```diff
@@ -230,6 +284,10 @@ $mobile: 500px;
         font-size: 20px;
         font-weight: 300;
         color: #74869b;
+
+        @media screen and (max-width: $mobile) {
+          display: none;
+        }
```

Comment:
Thank you for taking the time to iterate through all these mobile edge-cases.

Reply:
Holy moly this was a while ago, will merge now, sorry for the delay.